### PR TITLE
[AppConfig] Add --skip-keyvault for kv export

### PR DESCRIPTION
**Pull request by shenmuxiaosen**
**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by shenmuxiaosen** [comment]
@avanigupta Can you please take a look? Thanks.

File: src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py [hunk 1]
```diff
@@ -172,23 +173,18 @@ def export_config(cmd,
     src_kvs = __read_kv_from_config_store(
         cmd, name=name, connection_string=connection_string, key=key, label=label, prefix_to_remove=prefix)
 
+    if skip_keyvault:
+        src_kvs = [keyvaule for keyvaule in src_kvs if keyvaule.content_type != KeyVaultConstants.KEYVAULT_CONTENT_TYPE]
```
Comment (bim-msft) [inline src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py]: Seems `keyvaule` is a typo here, do you mean `keyvault` or `keyvalue`?

Comment (shenmuxiaosen) [reply src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py]: Typo. Should be keyvalue

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py [hunk 2]
```diff
@@ -360,6 +360,19 @@ def test_azconfig_import_export(self, resource_group, location):
             exported_kvs = json.load(json_file)
         assert imported_kvs == exported_kvs
 
+        # skip features while exporting
```
Comment (avanigupta) [inline src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py]: >features [](start = 15, length = 8)

nit: key vault ref

Comment (shenmuxiaosen) [reply src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py]: fixed.

**Comment by shenmuxiaosen** [comment]
@bim-msft Can you merge this pr if everything looks good? Thanks.

**Pull request merged**